stream_alert_cli/manage_lambda/deploy.py

```diff
@@ -149,5 +156,14 @@ def deploy(options, config):
     if not terraform_generate(config=config):
         return
 
+    if 'athena' in processors:
+        # create the streamalerts table since terraform does not support this
+        # See: https://github.com/terraform-providers/terraform-provider-aws/issues/1486
+        alerts_bucket = '{}.streamalerts'.format(config['global']['account']['prefix'])
+
+        athena_opts = namedtuple('AthenaOptions', ['bucket', 'refresh_type'])
+        opts = athena_opts(alerts_bucket, 'add_hive_partition')
+        athena_create_table(opts, 'alerts', config)
```

Comment:
I was under the impression we were revisiting table creation on every deploy per the last PR convo, is that not the case?